.goreleaser.yml

```diff
@@ -0,0 +1,6 @@
+before:
+  hooks:
+    - go mod tidy
```

Comment:
Very glad you asked this because I only decided to include this hook because the GitHub CLI also includes this hook, but looking more closely I think I misunderstood the behavior: I thought this would error out if the modules were untidy, but in fact it just fixes them, so that is definitely not what we want...also, pretty sure we would have caught any untidiness in our now very robust CI, so in any case I will remove this ↪️ 🎣 s.